test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -202,6 +215,13 @@ func CreateMongoDBResource(mdb *mdbv1.MongoDBCommunity, ctx *e2eutil.Context) fu
 	}
 }
 
+func GetConnectionStringForUser(mdb mdbv1.MongoDBCommunity, username string, database string) string {
```

Comment:
[q] why are we passing username and database here? can't we directly pass a scram user?

Reply:
passing now the scram user (since I created that method inside the user)